docker-compose.full.yml

```diff
@@ -122,6 +123,8 @@ services:
     extends:
       file: docker-services.yml
       service: db
+    volumes:
+      - ../fare_shared_data/fare_db:/backup_fare_db
```

Comment:
Anyway, I don't really like this. 
Why don't we:
1. declare a volume in the docker-compose.fare.yml file, call it "backups"
2. create a dump locally on the host
3. `docker cp` the dump from the host to the volume
4. docker exec che restoring command

Reply:
Check this resource for example: https://simkimsia.com/how-to-restore-database-dumps-for-postgres-in-docker-container/

Reply:
if i have understood correctly, i need to create a /backups volume for every container (ES, db and web-ui) and then when i restore the data i have only to do a "docker cp" command copying the files from the host to the container?

Reply:
Yes, for the db that makes sense because you will have a dump with pgdump. For the other images I believe this approach does not work. For the learning objects data we just need to backup the volume (e.g., rsync it to a safe folder) and for the ES one I don't have a proper way to do it, I leave the floor to you

Reply:
Since i store the dump of the database in the host and i make the restore from the dump of the host, create a backup in the db container could be useless. For the ES and files past i can use a `docker cp` command, from the containers to the host and from host to container for the restore. What do you think?